microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+* Non-critical CSS code used to render parts of the page that aren't visible when the page loads.
 
-Now, remove the references to these files from the code and audit the page again:
+    For example, the `/travel-site/assets/desktop.css` file is only needed when the viewport is larger than `665px`.
 
-1. Back in the editor tab, open `template.html`.
+To automatically split your CSS code this way, you can use the _Critical_ tool. To learn more, see the [Critical project repository](https://github.com/addyosmani/critical) at GitHub.
 
-1. Delete `<script src="/libs/lodash.js">` and `<script src="/libs/jquery.js"></script>`.
+You can then load your non-critical CSS code in a way to doesn't block the initial render of the page. To learn more, see [Defer non-critical CSS](https://web.dev/defer-non-critical-css/) at web.dev.
 
-1. Wait for the site to re-build and re-deploy.
+It's also good to minify your CSS code and remove unneeded whitespace characters and comments. To learn more, see [Minify CSS](https://web.dev/minify-css/) at web.dev.
 
-1. Audit the page again from the **Audits** tool.  Your overall score should have improved again.
+#### Reduce layout shift by setting explicit width and height on images
 
-   ![An Audits report after removing the render-blocking resources](../media/speed-glitch-tony-remix-updated-2-audits-performance.msft.png)
+In your **Lighthouse** report, the **Diagnostics** section suggests defining explicit `width` and `height` for image elements. Click on **Image elements do not have explicit width and height**  to display more information:
 
-#### Optimizing the Critical Rendering Path in the real-world
+![The Lighthouse diagnostics section about setting explicit width and height to images](./images/explicit-image-size.png)
 
-The **Critical Rendering Path** refers to the code that you need to load a page.  In general, speed up page load by only shipping critical code during the page load, and then lazy-loading everything else.
+The report says that the image at the top of the webpage doesn't have an explicit `width` and `height`, which can cause layout shifts.
 
-<!-- [Critical Rendering Path](/web/fundamentals/performance/critical-rendering-path/) -->
+Layout shifts occur when portions of a webpage initially appear in one place but move to another position during the load of the page. For example, when an image loads, the browser doesn't know how much space to reserve for it until the image is fully loaded.
 
-*  It's unlikely that you can find scripts that you can remove outright, but you might find many scripts that you don't need to request during the page load, which can instead be requested asynchronously.  <!-- See [Using async or defer](/web/fundamentals/performance/optimizing-content-efficiency/loading-third-party-javascript/#use_async_or_defer). -->
+To prevent layout shifts caused by loading images on a webpage:
```

Comment:
```suggestion
To prevent layout shifts caused by loading images on a webpage, do either of the following:
```
standard construction enables parallelism of list items